src/www/containers/landingpage/LandingPage.js

```diff
@@ -0,0 +1,173 @@
+              bgImage={bgkidimg}
+              bgImageAlt="Backgroundimage"
+              strength={150}
+            >
+              <div className="panel">
+                <div className="card">
+                  <p className="boxphrase">{t("landingpage.volunteer")}</p>
+                  <img
+                    className="boximg"
+                    alt="volunteer"
+                    src={"./img/volunteer.svg"}
+                  />
+                  <p className="boxdesc">{t("landingpage.volunteerdesc")}</p>
+                </div>
+                <div className="card">
+                  <p className="boxphrase">{t("landingpage.organize")}</p>
+                  <img
+                    className="boximg"
+                    alt="organize"
+                    src={"./img/organizer.svg"}
+                  />
+                  <p className="boxdesc">{t("landingpage.organizedesc")}</p>
+                </div>
+              </div>
+            </Parallax>
+          </div>
+
+          <div className="thirdpage">
+            <p className="phrase">{t("landingpage.findactions")}</p>
+            <Parallax
+              blur={0.5}
+              className="parallaxbg"
+              bgImage={bgkidimg}
+              bgImageAlt="Backgroundimage"
+              strength={150}
+            >
+              <div className="panel">
+                {(window.onresize = showCategories(categoriesList))}
+              </div>
+            </Parallax>
+          </div>
+        </div>
+      );
+    }
+  }
+);
+
+function showCategories(categoriesList) {
+  return Object.keys(categoriesList).map((key, index) => {
+    if (index < 3) {
+      return (
+        <div className="card">
+          <div className="box">
+            <img
+              className="boximg"
+              alt="category_img"
+              src={categoriesList[key].image}
+            />
+            <p className="category">{categoriesList[key].name}</p>
+          </div>
+          <p className="boxdesc">{categoriesList[key].desc}</p>
+        </div>
+      );
+    } else if (index < 4) {
+      return (
+        <div className="card card-4">
+          <div className="box">
+            <img
+              className="boximg"
+              alt="category_img"
+              src={categoriesList[key].image}
+            />
+            <p className="category">{categoriesList[key].name}</p>
+          </div>
+          <p className="boxdesc">{categoriesList[key].desc}</p>
+        </div>
+      );
+    } else if (index < 6) {
+      return (
+        <div className="card card-6">
+          <div className="box">
+            <img
+              className="boximg"
+              alt="category_img"
+              src={categoriesList[key].image}
+            />
+            <p className="category">{categoriesList[key].name}</p>
+          </div>
+          <p className="boxdesc">{categoriesList[key].desc}</p>
+        </div>
+      );
+    }
```

Comment:
Add an else case with `return null` since we get the following warning:
```
Line 128:55:  Expected to return a value at the end of arrow function  array-callback-return
```